ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -0,0 +1,104 @@
+package project
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseV3WithOneFile(t *testing.T) {
+	// set up file
+	numOfServices := 2
+	composeFileString := `version: '3'
+services:
+  wordpress:
+    image: wordpress
+    ports: ["80:80"]
+  mysql:
+    image: mysql`
+
+	tmpfile, err := ioutil.TempFile("", "test")
+	if err != nil {
+		t.Fatal("Unexpected error in creating test file: ", err)
+	}
+	defer os.Remove(tmpfile.Name())
+
+	if _, err := tmpfile.Write([]byte(composeFileString)); err != nil {
+		t.Fatal("Unexpected error writing to test file: ", err)
+	}
+	if err := tmpfile.Close(); err != nil {
+		t.Fatal("Unexpected error closing test file: ", err)
+	}
+
+	// add files to projects
+	project := setupTestProject(t)
+	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile.Name())
+
+	// assert number of container configs matches expected number of services
+	// TODO: assert containerConfig content after conversion implemented
+	conConfigResult, err := project.parseV3()
+	if err != nil {
+		t.Fatal("Unexpected error parsing file: ", err)
+	}
+	assert.Equal(t, numOfServices, len(*conConfigResult))
+}
+
+func TestParseV3WithMultileFiles(t *testing.T) {
+	// set up files
+	numOfServices := 3
+	firstFileString := `version: '3'
```

Comment:
Nittiest of nits: given you name the files `tmpfile1` and `tmpile2`, perhaps `filestring1` and `filestring2` would help make things less verbose/more consistent?